addons/mail/static/src/components/chat_window/chat_window.xml

```diff
@@ -25,7 +25,7 @@
                         <ThreadViewer
                             class="o_ChatWindow_thread"
                             composerAttachmentsDetailsMode="'card'"
-                            hasComposer="chatWindow.thread.model !== 'mail.box'"
+                            hasComposer="chatWindow.thread.model !== 'mail.box' and (!chatWindow.thread.mass_mailing or env.messaging.device.isMobile)"
```

Comment:
Mailing icon is "missing" from chat window header.
I'm not sure if it was originally there, but the lack of this icon makes it unclear why there is no composer.

Reply:
Don't think it was the case before, just added it:
![chat-window-mailing-channel-icon](https://user-images.githubusercontent.com/6569390/86945977-02fb8480-c14a-11ea-86b4-4dbcccf2adff.png)